prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/Quantity.yml

```diff
@@ -0,0 +1,46 @@
+code:
+  condition: $identifier NOT_NULL && $nameOfCodingSystem NOT_NULL
+  type: STRING
+  expressionType: HL7Spec
+  valueOf: CWE.1
+  vars:
+    identifier: CWE.1
+    nameOfCodingSystem: CWE.3
+
+unit:
+  generateList: true
+  expressionType: nested
+  condition:
+  vars:
+    identifier: CWE.1
+    text: CWE.2
+  expressions:
+    - condition: $text NULL && $identifier NOT_NULL
+      type: STRING
+      valueOf: CWE.1
+      expressionType: HL7Spec
+    - condition: $text NOT_NULL
+      type: STRING
+      valueOf: CWE.2
+      expressionType: HL7Spec
+
+system:
+  condition: $identifier NOT_NULL && $nameOfCodingSystem NOT_NULL
+  type: SYSTEM_URL
```

Comment:
I'm not sure if we've done this in other places... wouldn't this mean we would only populate it if is in our URL mappings?

Reply:
yes that's true. We do this in the current/old Coding mapping and in the new CodeableConcept mapping. But I don't rely on it for the FHIR --> HL7 mapping.  I added "L" for local codes to the files in the CodeableConcept PR.  we do this in

Reply:
That makes sense to me, I'll put on the list of things we should think about for FHIR receivers